koala/cogs/base/core.py

```diff
@@ -96,3 +97,128 @@ async def activity_set_current_scheduled(bot: Bot, **kwargs):
         await bot.change_presence(activity=new_activity)
         logger.info("Auto changing bot presence: %s" % new_activity)
         current_activity = new_activity
+
+
+async def ping(bot: Bot):
+    """
+    Returns the ping of the bot
+    :param bot:
+    :return:
+    """
+    return f"Pong! {round(bot.latency * 1000)}ms"
+
+
+def support_link():
+    """
+    Returns the link for KoalaBot Support server
+    :return:
+    """
+    return f"Join our support server for more help! https://discord.gg/5etEjVd"
+
+
+async def purge(bot: Bot, channel_id, amount):
+    """
+    Purges a number of messages from the channel
+    :param channel:
+    :param amount:
+    :return:
+    """
+    channel = bot.get_channel(channel_id)
+    return await channel.purge(amount + 1)
+
+
+async def load_cog(bot: Bot, extension, package):
+    """
+    Loads a cog from the cogs folder
+    :param extension:
+    :param package:
+    :return:
+    """
+    bot.load_extension("."+extension, package=package)
+    return f'{extension} Cog Loaded'
+
+
+async def unload_cog(bot: Bot, extension, package):
+    """
+    Unloads a cog from the cogs folder
+    :param extension:
+    :param package:
+    :return:
+    """
+    if extension == "base" or extension == "BaseCog":
+        return "Sorry, you can't unload the base cog"
+    else:
+        bot.unload_extension("."+extension, package=package)
+        return f'{extension} Cog Unloaded'
+
+
+async def enable_extension(bot: Bot, guild_id, koala_extension):
+    """
+    Enables a koala extension
+    :param guild_id:
+    :param koala_extension:
+    :return:
+    """
+    if koala_extension.lower() in ["all"]:
+        available_extensions = get_all_available_guild_extensions(guild_id)
```

Comment:
Any core method that accesses the database should use `@assign_session` and pass the session between the core method and the database method using kwargs, please see  `def activity_schedule():`

This change will be needed on ALL methods that access the database from core